### PR TITLE
feat: Upgrade to rxjs and angular libs to v6

### PR DESCRIPTION
**Pull request by jrollin**
Following update of angular 6 and rxjs 6 
(see also #947)

* update package.json deps
* remove tslint blacklisting rule to match new rxjs import

### CODE REVIEW

**Comment by trimox** [comment]
@jrollin Thank you for a great contribution! 💎 

I'll go through this and nitpick for style and what not, (it's never personal, just want to maintain consistency), and then we can get this merged in.

File: src/lib/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@angular-mdc/theme": "0.35.2"
   },
   "peerDependencies": {
-    "@angular/core": ">=5.0.0",
-    "@angular/common": ">=5.0.0"
+    "@angular/core": ">=6.0.0",
```
Comment (trimox) [inline src/lib/package.json]: 👍

**Comment by trimox** [comment]
@jrollin Internal testing results looked great! 

Once the general comment above is resolved, I'll re-review and merge in 😄

**Comment by jrollin** [comment]
@trimox great !  I did the update 😄

**Comment by trimox** [comment]
LGTM. Thanks!

**Pull request merged**